about.html

```diff
@@ -0,0 +1,156 @@
+              <a class="nav-link" aria-current="page"
+                href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="index.html#work-section">Services</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="index.html#team-section">Teams</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link btn-outline" href="#"> Contact Us</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </header>
+
+  <section id="about-section">
+    <div class="wrapper-about-open">
+      <a id="about-open" class="open-menu" href="#"><i class="fa fa-bars fa-4x"
+          aria-hidden="true"></i></a>
+    </div>
+    <div class="about-header">
+      <h2 class="type-1">"Our Service is Real!"</h2>
+      <h1 class="about-heading">BAMEN, SERVICE, TEACH AND DESIGN</h1>
+      <div class=wrapper-text>
+        <p class="about-desc">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptates
+          veniam sunt. Lorem ipsum dolor sit amet.
+        </p>
+        <p class="about-desc">Lorem ipsum dolor sit amet consectetur adipisicing
+          elit. Lorem ipsum dolor sit amet.
+        </p>
+        <p class="about-desc">Lorem ipsum dolor sit amet consectetur,
+          adipisicing elit. Aperiam Lorem ipsum dolor sit.
+          iste dolore rerum. Lorem ipsum dolor sit, amet consectetur adipisicing
+          elit. Perferendis, vero.</p>
+      </div>
+      <p class="further-contact">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+        Exercitationem expedita sapiente libero fugiat aut sit magni totam
+        dolore labore veritatis?
+      </p>
+
+      <div class="wrapper-about">
+        <h2 class="pic-title">Lorem ipsum dolor sit</h2>
+        <div class="about-border"></div>
+        <p class="pic-desc">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum
+          accusamus quae animi.
+        </p>
+
+        <div class="logo-container">
+          <img class="logo-pic" src="assets/images/pictures/logo.png"
+            alt="logo picture" />
+        </div>
+      </div>
+    </div>
+    <hr class="separator" />
+    <div class="previous-works">
+      <div class="experiance-1">
+        <div class="image-overlay"></div>
+        <img class="work-logo" src="assets/images/pictures/two-person.png"
+          alt=" two persons standing" />
+        <div class="work-details">
+          <span class="year">2021</span>
+          <p class="work-para">Lorem ipsum dolor</p>
+        </div>
+      </div>
+      <div class="experiance-2">
+        <div class="image-overlay"></div>
+        <img src="assets/images/pictures/group.png" alt=" a group of people" />
+        <div class="work-details">
+          <span class="year">2020</span>
+          <p class="work-para">Lorem ipsum dolor sit amet.</p>
+        </div>
```

Comment:
- [ ] I have noticed that the background image of this section is breaking page on mobile screens:
![image](https://user-images.githubusercontent.com/77038610/152215727-c8a5b61e-a790-432f-8507-b00d0f5668a4.png)
This can be easily fixed if you give `width: 100%;` property to background image:

https://user-images.githubusercontent.com/77038610/152216039-32a25944-b71f-4d0e-99ca-5d6d945ba4f0.mp4